ognjen/pay-pal/src/layouts/main.jsx

```diff
@@ -0,0 +1,7 @@
+import TopNavBar from "../components/TopNavBar"
+
+export default function Layout ({children}) {
```

Comment:
Read above in PR review what layout should do, or check out with @NanaAV or @marinajokic if it's unclear